dao-contracts/src/reputation.rs

```diff
@@ -62,8 +61,10 @@ pub trait ReputationContractInterface {
     /// It throws [`NotAnOwner`](casper_dao_utils::Error::NotAnOwner) if caller
     /// is not the current owner.
     ///
-    /// It emits [`OwnerChanged`](casper_dao_modules::events::OwnerChanged),
-    /// [`AddedToWhitelist`](casper_dao_modules::events::AddedToWhitelist) events.
+    /// It emits [`OwnerChanged`](casper_dao_utils::owner::events::OwnerChanged),
+    /// [`AddedToWhitelist`](casper_dao_utils::whitelist::events::AddedToWhitelist) and
+    /// [`RemovedFromWhitelist`](casper_dao_utils::whitelist::events::RemovedFromWhitelist)
```

Comment:
Does it? Currently, the old owner remains on the whitelist. So please update the doc or the implementation.